src/test/java/org/opensearch/security/dlic/rest/api/DashboardsInfoActionTest.java

```diff
@@ -24,39 +25,53 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
-import java.util.Arrays;
+
+import static org.opensearch.security.OpenSearchSecurityPlugin.LEGACY_OPENDISTRO_PREFIX;
+import static org.opensearch.security.OpenSearchSecurityPlugin.PLUGINS_PREFIX;
 
 @RunWith(Parameterized.class)
 public class DashboardsInfoActionTest extends AbstractRestApiUnitTest {
 
     private final String ENDPOINT;
-    private final String CONFIG_ENDPOINT;
+    private final String SECURITY_ENDPOINT;
+
+    public DashboardsInfoActionTest(String endpoint, String securityEndpoint){
 
-    public DashboardsInfoActionTest(String configEndpoint, String endpoint){
-        CONFIG_ENDPOINT = configEndpoint;
         ENDPOINT = endpoint;
+        SECURITY_ENDPOINT = securityEndpoint;
     }
 
     @Parameterized.Parameters
-    public static Iterable<Object[]> endpoints() {
+    public static Iterable<String[]> endpoints() {
         return Arrays.asList(new String[][] {
-                {"_opendistro/_security/kibanainfo", "/_opendistro/_security/api/securityconfigt"},
-                {"_security/_security/kibanainfo", "/_security/_security/api/securityconfigt"}
+                {LEGACY_OPENDISTRO_PREFIX + "/kibanainfo", LEGACY_OPENDISTRO_PREFIX + "/api/securityconfig"},
```

Comment:
I still think that `securityconfig` is secondary to the test and should not be part of the parameters. Whether test uses `LEGACY_OPENDISTRO_PREFIX + "/api/securityconfig"` or `PLUGINS_PREFIX + "/api/securityconfig` is not important (for the test). Equally `kibanainfo` can use `PLUGINS_PREFIX + "/api/securityconfig"` to update the config.

Reply:
So do you suggest to remove the test for `securityconfig`?

Reply:
Yes, `securityconfig` should be tested separately from DashboardsInfoActionTests. I mentioned that in one of my comments.

Reply:
Where do you suggest to put the unit test for `securityconfig`?

Reply:
We have unit tests for it in `securityConfigApiTest`, `securityHealthActionTest`, `securityInfoActionTest` and `tenantInfoActionTest`

Reply:
@afazel Please explain reason/goal/test case why there are calls to `securityconfig` in `SecurityHealthActionTest` and `SecurityInfoActionTest`? Compare it with `TenantInfoActionTest`. The actual test for `securityconfig` is done in `SecurityConfigApiTest`.

Reply:
@vrozov From what I see, they are exact same tests in different files, so I think we do not need that unit test in the `DashboardsInfoActionTest`, just unit tests for `kibanainfo` and `DashboardsInfo` should we sufficient.

Reply:
I removed the redundant unit tests for `securityconfig` from `SecurityHealthActionTest` and `SecurityInfoActionTest`. I did not remove it from `TenantInfoActionTest` as it did not introduce by my [previously merged PR](https://github.com/opensearch-project/security/pull/1172/files)